server/notification-providers/wecom.js

```diff
@@ -30,26 +31,50 @@ class WeCom extends NotificationProvider {
     /**
      * Generate the message to send
      * @param {object} heartbeatJSON Heartbeat details (For Up/Down only)
+     * @param {object} monitorJSON Monitor details
      * @param {string} msg General message
      * @returns {object} Message
      */
-    composeMessage(heartbeatJSON, msg) {
-        let title;
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === UP) {
-            title = "UptimeKuma Monitor Up";
-        }
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === DOWN) {
-            title = "UptimeKuma Monitor Down";
+    composeMessage(heartbeatJSON, monitorJSON, msg) {
+        let params = {};
+        if (heartbeatJSON != null) {
+            params = {
+                msgtype: "markdown",
+                markdown: {
+                    content: `## [${this.statusToString(
+                        heartbeatJSON["status"]
+                    )}] ${monitorJSON["name"]} \n> ${
+                        heartbeatJSON["msg"]
+                    } \n> Time (${heartbeatJSON["timezone"]}): ${
+                        heartbeatJSON["localDateTime"]
+                    }`,
+                },
+            };
+        } else {
+            params = {
+                msgtype: "text",
+                text: {
+                    content: msg,
+                },
+            };
         }
```

Comment:
Unclear if the indentation in this correction is 100% correct

```suggestion
        }
        return {
            msgtype: "text",
            text: {
                content: msg,
            },
        };
```

Reply:
it's automatic indented with prettier, and I also use this one for myself. So I would assume it's correct.